encord/orm/label_row.py

```diff
@@ -418,8 +421,8 @@ class LabelRowMetadataDTO(BaseDTO):
 
     priority: Optional[float] = None
     """Only available for not complete tasks"""
-    client_metadata: Optional[dict] = None
-    images_data: Optional[list] = None
+    client_metadata: Optional[Dict[str, Any]] = None
```

Comment:
What's on the left isn't strictly Python 3.8 compliant.
Any isn't strictly true as it should be dict[str, JSONSERIALIZABLE] but small step in right direction.